seed/static/seed/partials/column_settings.html

```diff
@@ -56,8 +56,12 @@ <h3 translate>MODIFYING_COLUMN_SETTINGS</h3>
                 <table id="column-table" class="table has_no_btm_margin">
                     <thead>
                         <tr>
-                            <th translate>Display Name</th>
-                            <th translate>Column Name</th>
+                            <th class="text-left" ng-click="toggle_display_name_order_sort()">
+                                <span>Display Name<i class="glyphicon glyphicon-sort"></i></span>
```

Comment:
I'm thinking we should make the icons all the same. I kind of like the `sort` icon you're using vs the `list` icon that existed before, but I don't have a strong opinion either way.

@axelstudios any thoughts?
<img width="975" alt="Screen Shot 2020-01-24 at 12 38 29 PM" src="https://user-images.githubusercontent.com/30608004/73098481-b9bebd80-3ea6-11ea-8308-8ba7784baa68.png">

Reply:
I agree with @adrian-lara -- that is the sort icon is a bit nicer!

Reply:
Same - I like _sort_